apps/versions/tests.py

```diff
@@ -465,6 +466,21 @@ def test_sign_files(self, sign_mock):
         self.version.sign_files()
         assert sign_mock.called
 
+    @mock.patch('lib.crypto.packaged.call_signing')
+    @mock.patch('lib.crypto.packaged.get_endpoint')
+    def test_sign_version_missing_files(self, get_endpoint, call_signing_mock):
+        """If one of the files is missing the others should still be signed."""
+        get_endpoint.return_value = 'endpoint'  # Fake an endpoint.
+        self.version.addon.update(type=amo.ADDON_EXTENSION)
+        # Current file doesn't have a file on disk. We create a file with one.
+        good_file = amo.tests.file_factory(version=self.version)
+        with amo.tests.copy_file('apps/files/fixtures/files/jetpack.xpi',
+                                 good_file.file_path):
+            self.version.sign_files()
+        # 'sign_file' got called twice, but only one was signed.
+        call_signing_mock.assert_called_once_with(
+            good_file.file_path, 'endpoint', good_file.version.addon.guid)
```

Comment:
There's already a file in the fixture, but it doesn't have a real file on the disk (it's just a db fixture).

That's why I'm adding another File with a file on disk. This is the one that needs to get signed (and the original one not having a file on disk shouldn't prevent that).